tests/python/frontend/tflite/test_forward.py

```diff
@@ -2127,7 +2127,10 @@ def test_all_unary_elemwise():
     if tf.__version__ < LooseVersion("2.6.1"):
         _test_forward_unary_elemwise(_test_rsqrt, negative=False, int_quant_dtype=tf.uint8)
     else:
-        _test_forward_unary_elemwise(_test_rsqrt, negative=False, int_quant_dtype=tf.int8)
+        # This test is flaky, see https://github.com/apache/tvm/issues/11567
```

Comment:
hey sorry i realized we documented this a bit: https://github.com/apache/tvm/blob/main/jenkins/README.md#dealing-with-flakiness

wdyt about splitting this test into pieces and marking this piece as xfail?

Reply:
done